db/transaction/transaction.go

```diff
@@ -59,8 +59,8 @@ type ResourceState struct {
 	Monitoring    string
 }
 
-//ListOptions specifies additional list related options.
-type ListOptions struct {
+//ViewOptions specifies additional list related options.
```

Comment:
BTW, how do you refactor naming? by hand? InteliJ has a nice and working rename tool which also renames usages in comments.